src/Security/Member.php

```diff
@@ -1705,6 +1705,16 @@ public function getHtmlEditorConfigForCMS()
         $currentName = '';
         $currentPriority = 0;
 
+        // If we don't have a custom config, no need to look in all groups
+        $editorConfigMap = HTMLEditorConfig::get_available_configs_map();
+        $editorConfigCount = count($editorConfigMap);
+        if ($editorConfigCount === 0) {
+            return 'cms';
+        }
+        if ($editorConfigCount === 1) {
+            return key($editorConfigMap);
+        }
+
         foreach ($this->Groups() as $group) {
```

Comment:
If we really wanted to be fancy, we could add a filter here `$this->Groups()->filter(['HtmlEditorConfig:not' => null]) - but the benefit of that would be super low so not worth holding up merging just for that. If you're keen on that I'd be happen to merge a separate PR for that change on its own.